standalone/src/models/order/payment-info/fields-config.ts

```diff
@@ -0,0 +1,21 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { Payment } from '@/models/payment';
+import type { TPaymentInfoGraphql, TPaymentInfoRest } from './types';
+
+const commonFieldsConfig = {
+  payments: [fake(() => Payment.random())],
```

Comment:
The `payments` property can't be shared.
As you can see in the [REST docs](https://docs.commercetools.com/api/projects/orders#paymentinfo), it is expected to be an array of references but in GraphQL is it should be actual `Payment` objects.
<img width="238" height="234" alt="image" src="https://github.com/user-attachments/assets/04175dd7-2f88-426f-bdf0-61f18f81da1d" />

Reply:
Fixed here: 006156f20c79345520669a13157396909531243d